themes/default/content/docs/get-started/gcp/deploy-changes.md

```diff
@@ -296,10 +296,11 @@ bucketObject, err := storage.NewBucketObject(ctx, "index.html", &storage.BucketO
     ContentType: pulumi.String("text/html"),
     Source:      pulumi.NewFileAsset("index.html"),
 })
-bucketEndpoint := pulumi.Sprintf("http://storage.googleapis.com/%s/%s", bucket.Name, bucketObject.Name)
 if err != nil {
     return err
 }
+
+bucketEndpoint := pulumi.Sprintf("http://storage.googleapis.com/%s/%s", bucket.Name, bucketObject.Name)
```

Comment:
Would you mind moving this to the next code block?
The text flow is then:

- change the content type of index.html to serve it as HTML, and here's a code block that does just that
- export the resulting bucket endpoint URL, and here's a code block that builds the endpoint URL with Sprintf, and exports it